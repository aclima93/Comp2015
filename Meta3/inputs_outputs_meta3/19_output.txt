Line 6, col 1: Symbol gcd3 not defined
