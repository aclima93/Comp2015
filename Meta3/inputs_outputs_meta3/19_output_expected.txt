===== Outer Symbol Table =====
boolean	_type_	constant	_boolean_
integer	_type_	constant	_integer_
real	_type_	constant	_real_
false	_boolean_	constant	_false_
true	_boolean_	constant	_true_
paramcount	_function_
program	_program_

===== Function Symbol Table =====
paramcount	_integer_	return

===== Program Symbol Table =====
spj	_function_

===== Function Symbol Table =====
spj	_integer_	return
w	_integer_	param
coiso	_integer_
coiso2	_integer_
